docs/participate/community_resources/community-build-tools.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Community Build Tools
+sidebar_position: 2
+description: essential info to file systems. partitioning, and mounting.
```

Comment:
Should this be updated? I'm not sure where it's used?

Reply:
This is used for indexing, it's better to update it. Thanks for catching it!